.github/workflows/reusable_swap_functional_tests.yml

```
@@ -177,6 +186,9 @@ jobs:
         - name: cardano
           repo: ${{ inputs.repo_for_cardano }}
           branch: ${{ inputs.branch_for_cardano }}
+        - name: cosmos
```

this will be used as `COIN=` value
```suggestion
        - name: ATOM
```